xml/System.Text.Encodings.Web/HtmlEncoder.xml

```diff
@@ -13,8 +13,8 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents a type used to do HTML encoding.</summary>
```

Comment:
It can't represent a type, since it is a type. So better, "Represents an HTML character encoding." This comment applies to the other types as well.